pyproject.toml

```diff
@@ -29,7 +29,7 @@ dependencies = [
     "pika >= 1.2.0",
     "dataset",
     "pymongo > 3.0",
-    "sdx-pce @ git+https://github.com/atlanticwave-sdx/pce@3.1.0.dev1",
+    "sdx-pce @ git+https://github.com/atlanticwave-sdx/pce@main",
```

Comment:
this is temporary while we wait for PR https://github.com/atlanticwave-sdx/pce/pull/274 goes on the review/approval process.

Reply:
Although PR https://github.com/atlanticwave-sdx/pce/pull/274 got merged, there was a action to re-tag 3.1.0.dev1, so we will have to re-work this one later.